pages/_app.tsx

```diff
@@ -153,6 +154,7 @@ export default function PlanetWeb({ Component, pageProps, err }: any) {
       >
         <ThemeProvider>
           <CssBaseline />
+          <UserPropsProvider>
```

Comment:
@sarveshpro can we be sure the user handling in this abstracted HOC does not interfere with any other handling in our code e.g. https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/ec938bc0d35d5ac659fc1710bebb9f65446b2bb2/pages/t/%5Bid%5D.tsx#L90-L112 or https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/ec938bc0d35d5ac659fc1710bebb9f65446b2bb2/src/utils/auth0/userInfo.ts#L7-L46

Reply:
At least this change cost us one additional API call to fetch the user's profile. 

- [ ] We already have a duplication at `/login` following even by a third call when redirecting to the user's profile `/t/user` - now this is calling the profile API four(!) times.

<img width="1683" alt="Bildschirmfoto 2021-05-03 um 08 23 14" src="https://user-images.githubusercontent.com/1532418/116847075-1b152680-abea-11eb-9764-2e2d83393b48.png">

- [ ] The duplication now is calling the profile API call twice being logged in when opening the user's profile.

<img width="819" alt="Bildschirmfoto 2021-05-03 um 08 24 06" src="https://user-images.githubusercontent.com/1532418/116847097-22d4cb00-abea-11eb-922a-da626850fe1f.png">